src/api/todos.ts

```diff
@@ -0,0 +1,36 @@
+import { Todo } from '../types/Todo';
+import { client } from '../utils/fetchClient';
+
+export const getTodos = (userId: number) => {
```

Comment:
For consistency and better readability, consider adding a return type `Promise<Todo[]>` for the `getTodos` function. 😉